Readme.md

```diff
@@ -57,33 +57,38 @@ $ kubectl apply -f manifests/ # This command sometimes may need to be done twice
 
 If you get an error from applying the manifests, run the `make deploy` or `kubectl apply -f manifests/` again. Sometimes the resources required to apply the CRDs are not deployed yet.
 
-## Customizing for K3s
+## Quickstart (K3s)
 
-To have your [K3s](https://github.com/rancher/k3s) cluster and the monitoring stack on it, follow the steps:
+To deploy the monitoring stack on your K3s cluster, there are three parameters that need to be configured in the  `vars.jsonnet` file:
 
+1. Set `k3s.enabled` to `true`.
+2. Change your K3s master node IP(your VM or host IP) on `k3s.master_ip`.
+3. Edit `suffixDomain` to have your node IP with the `.nip.io` suffix. This will be your ingress URL suffix.
+4. Set _traefikExporter_ `enabled` parameter to `true` to collect Traefik metrics and deploy dashboard.
+
+After changing these values, to simply deploy the stack, run:
 ```bash
-# Download K3s binary
-wget https://github.com/rancher/k3s/releases/download/`curl -s https://api.github.com/repos/rancher/k3s/releases/latest | grep -oP '"tag_name": "\K(.*)(?=")'`/k3s && chmod +x k3s
+$ make deploy
 
-# Move to your path
-sudo mv k3s /usr/local/bin
+# Or manually:
 
-# Start K3s
-sudo k3s server &
+$ kubectl apply -f manifests/
+
+# It can take a few seconds for the above 'create manifests' command to fully create the following resources, so verify the resources are ready before proceeding.
+$ until kubectl get customresourcedefinitions servicemonitors.monitoring.coreos.com ; do date; sleep 1; echo ""; done
+$ until kubectl get servicemonitors --all-namespaces ; do date; sleep 1; echo ""; done
+
+$ kubectl apply -f manifests/ # This command sometimes may need to be done twice (to workaround a race condition).
 ```
 
-Now to deploy the monitoring stack on your K3s cluster, there are three parameters to be configured on `vars.jsonnet`:
 
-1. Set `k3s.enabled` to `true`.
-2. Change your K3s master node IP(your VM or host IP) on `k3s.master_ip`.
-3. Edit `suffixDomain` to have your node IP with the `.nip.io` suffix. This will be your ingress URL suffix.
-4. Set _traefikExporter_ `enabled` parameter to `true` to collect Traefik metrics and deploy dashboard.
+If you get an error from applying the manifests, run the `make deploy` or `kubectl apply -f manifests/` again. Sometimes the resources required to apply the CRDs are not deployed yet.
 
-After changing these values, run `make` to build the manifests and `k3s kubectl apply -f manifests/` to apply the stack to your cluster. In case of errors on some resources, re-run the command.
+## Ingress 
 
 Now you can open the applications:
 
-To list the created ingresses, run `k3s kubectl get ingress --all-namespaces`.
+To list the created ingresses, run `kubectl get ingress --all-namespaces`, you will notice they need to be updated to include `your_node_ip`.
```

Comment:
If you edit `vars.jsonnet` adding the correct IP for your infress before running `make` and deploying the manifests, the ingresses are created correctly.
No need to update them after deployment. Maybe clarify this.